asabiris/errors.py

```diff
@@ -29,4 +29,13 @@ def get_i18n_name(self):
 	TEMPLATE_NOT_FOUND = 1004
 	TEMPLATE_SYNTAX_ERROR = 1005
 	JINJA2_ERROR = 1006
-	GENERAL_ERROR = 1007
+	SERVER_ERROR = 1007
+	INVALID_SERVICE_CONFIGURATION = 1008
+	JINJA2_RENDERING_ERROR = 1009
```

Comment:
What is the difference between JINJA2_RENDERING_ERROR and JINJA2_ERROR?

Reply:
I renamed it from JINJA2_ERROR  to RENDERING_ERROR to avoid confusion
 
 
**RENDERING_ERROR**: This error code encompasses any errors encountered during the template rendering process. While there may be other operations or processes involved in the function beyond just rendering, this error code is used as a general indicator that something went wrong during the overall rendering process.

**JINJA2_ERROR**: This error code specifically indicates that there was an error related to the Jinja2 templating engine. Jinja2 is a powerful templating engine used in Python applications. When this error occurs, it suggests that the issue lies within the Jinja2 processing itself, such as syntax errors or other Jinja2-specific problems.